catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageAttribute.java

```diff
@@ -159,13 +156,6 @@
      * appropriate
      */
 
-    public static final ImageAttribute IMAGE_DATE_AND_TIME_ATTRIBUTE =
-            new ImageAttribute(ddf.catalog.data.types.DateTime.START,
-                    "IDATIM",
-                    segment -> convertNitfDate(segment.getImageDateTime()),
-                    new DateTimeAttributes().getAttributeDescriptor(ddf.catalog.data.types.DateTime.START),
-                    IMAGE_DATE_AND_TIME);
```

Comment:
`IMAGE_DATE_AND_TIME_ATTRIBUTE` should not have been a normalized attribute because this caused the `dateTime.start` to have a value for each image segment, but `dateTime.start` should not be multi-valued.